### DISCUSSION TITLE
Minor wiki dead link issue

### DISCUSSION THREAD

**Discussion by stdedos** [uBlockOrigin/uBlock-issues#2961] [Ideas] [answered]
In https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#html-filters, "Also see: [remove-node-text](https://github.com/gorhill/uBlock/wiki/Static-filter-syntax#remove-node-textjs-)" should lead to https://github.com/gorhill/uBlock/wiki/Resources-Library#rmntjs- instead

**Comment by ghost** [comment, marked as answer]
Thanks. Fixed in draft https://github.com/uBlockOrigin/uBlock-issues/wiki/Static-filter-syntax/_compare/cbc0d806c795e4bf8cbc7f27e512f15ea8b1c743...9e6d0b2b9cff77b9767a37d42fbd8d8f4a50f5c1

**Reply by stdedos** [reply]
_Oh, it's editable when you sign in_ 🙈

**Reply by ghost** [reply]
This version is editable:

https://github.com/uBlockOrigin/uBlock-issues/wiki 

This one is not

https://github.com/gorhill/uBlock/wiki

**Reply by stdedos** [reply]
The plot thickens 🙃 

So ... are they "the same wiki"? 😕 
Should I "just make changes" in the future there?

**Reply by ghost** [reply]
Changes from the editable one are reviewed and pulled into the official one.
